doc/troubleshooting.md

```diff
@@ -14,4 +14,4 @@ Run:
 
 ### Other Problems
 
-Please ask for help on the Slack channel. Join at [slack.cordova.io](http://slack.cordova.io)
\ No newline at end of file
+Please ask for help on the Slack channel. Join at [slack.cordova.io](http://slack.cordova.io)
```

Comment:
GH is showing that there is a diff on this line but I see no visable difference.

Also, this URL does not work with either HTTP or HTTPS but no changed required.